lib/secureAccessMethods.js

```diff
@@ -174,10 +174,28 @@ function createSecure(user, options) {
         });
 }
 
+/**
+ * allows a protected method to be called
+ * @param  {string} method   protected method to call
+ * @param  {object} options  options to call function with
+ * @return {Promise}         Promise that the called function returns
+ */
+function bypass(method, options) {
+    const self = this;
+    const shield = self.constructor.shield;
+
+    if (!_.includes(shield.protectedMethods, method)) {
+        return Promise.reject(new Error('no such protected method'));
+    }
+    const protectedMethod = `_${method}`;
+    return shield[protectedMethod].call(self, options);
```

Comment:
I just copied Dylan's calls above ie: `return shield._fetchAll.call(self, options)...`, but you're probably right